File: TrabajoI.C/C#/interfaz/ClaseE.cs
public class ClaseE : IOperable
{
    private double precio;

    public E(double precio)  // Tipo repetido
    {
        this.precio = precio;
        Console.WriteLine($"Constructor con parámetros en E: {precio}");
    }

    public void Operar()
    {
        Console.WriteLine("Operando en E");
    }
}

File: TrabajoI.C/Java/interfaz/ClaseE.java
package interfaz;

public class ClaseE implements Operable {
    double precio;

    public ClaseE(double precio) { // Constructor con parámetros (repetido)
        this.precio = precio;
        System.out.println("Constructor con parámetros en E: " + precio);
    }

    @Override
    public void operar() {
        System.out.println("Operando en E");
    }
}
